intel_extension_for_transformers/llm/library/jblas/jblas/jit_blas_epilogue.h

```diff
@@ -35,20 +39,24 @@ class AccumulatorWriteBack {
                      const int N, const Param& _param, Eltops... ops) {
     auto COffset = M_offset * _param.ldc + N_offset;
     auto cptr = _param.C + COffset;
-    bool constexpr Valid = !std::is_same<_DST_T, jblas::utils::bf16>::value ? true : std::is_same<_SRC_T, float>::value;
+    bool constexpr Valid = !std::is_same<DType, utils::bf16>::value || std::is_same<SType, float>::value;
     static_assert(Valid, "fp32 to bf16 conversion only.");
-    if (std::is_same<_DST_T, jblas::utils::bf16>::value) {
+    if (std::is_same<DType, utils::bf16>::value) {
```

Comment:
should we add constexpr before any condition judge in templates

Reply:
yes, we should add constexpr as a standard behavior